pkg/build/builder/daemonless.go

```diff
@@ -201,9 +201,11 @@ func pullDaemonlessImage(sc types.SystemContext, store storage.Store, imageName
 		BlobDirectory: blobCacheDirectory,
 	}
 	_, err = buildah.Pull(context.TODO(), "docker://"+imageName, options)
-	log.V(0).Infof("Finished pulling image %s", imageName)
+	if err == nil {
+		log.V(2).Infof("Finished pulling image %q", imageName)
+	}
```

Comment:
```suggestion
	} else {
	  log.Errorf("Error pulling image %q: %s",imageName,err.Error())
	}
```

Reply:
oh wait you suggested change is for that gap

Reply:
there is not `log.Errorf` ... just going to keep V(2) as we may be able to ingore this / recover on a retry